ui/admin/tests/acceptance/storage-buckets/create-test.js

```diff
@@ -105,6 +107,44 @@ module('Acceptance | storage-buckets | create', function (hooks) {
     assert.strictEqual(getStorageBucketCount(), storageBucketCount + 1);
   });
 
+  test('users can create a new storage bucket with dynamic credentials', async function (assert) {
+    assert.expect(3);
+    const storageBucketCount = getStorageBucketCount();
+    await visit(urls.storageBuckets);
+
+    await click(`[href="${urls.newStorageBucket}"]`);
+    await fillIn(NAME_FIELD_SELECTOR, NAME_FIELD_TEXT);
+    assert.dom('.hds-form-radio-card').exists({ count: 2 });
+
+    await click(DYNAMIC_CREDENTIAL_SELECTOR);
+    await fillIn(ROLE_ARN_SELECTOR, 'test-arn-id');
+    await click(SAVE_BTN_SELECTOR);
+    const storageBucket = this.server.schema.storageBuckets.findBy({
+      name: NAME_FIELD_TEXT,
+    });
+    assert.strictEqual(storageBucket.name, NAME_FIELD_TEXT);
+    assert.strictEqual(getStorageBucketCount(), storageBucketCount + 1);
```

Comment:
one small thing: i think a assert can be added here to make sure that `storageBucket.credentialType` is equal to `TYPE_CREDENTIAL_STATIC`

Reply:
ah yes, good point, I'll include these

Reply:
@lisbet-alvarez - So, the model properties(shown in the desc screenshot) aren't exposed here. However, the unit test on the SB model covers the logic for `credentialtype`.  I added some extra UI tests to ensure we get the correct cred type.

Reply:
ahh okay makes sense, thanks for looking into that!